src/main/java/Drink.java

```diff
@@ -0,0 +1,5 @@
+public class Drink {
+    String name;
+    double price;
+    boolean alcohol;
+}
```

Comment:
Drink składa się z trzech składników. Ta klasa nie odzwierciedla tego wymagania.
W rozwiązaniu powinny być trzy klasy:
- Drink
- Ingredient
- jakaś klasa z metodą main, w której utworzysz drinki (obiekty)

Spójrz na lekcję o agregacji i kompozycji. Np. to ćwiczenie:
https://app.javastart.pl/kurs/java/java-kompozycja/cwiczenie/java-kompozycja-cwiczenie-1
Osoba (Person), ma adres zamieszkania i zameldowania. Tutaj będzie podobnie z relacją drink<>składniki